src/Microsoft.Data.Analysis/StringDataFrameColumn.cs

```diff
@@ -169,12 +170,36 @@ public IEnumerator<string> GetEnumerator()
 
         public override DataFrameColumn Filter<U>(U lower, U upper) => throw new NotSupportedException();
 
+        public new StringDataFrameColumnRollingWindow Rolling(int windowSize)
+        {
+            return new StringDataFrameColumnRollingWindow(windowSize, this);
```

Comment:
What about ArrowStringDataFrameColumn? Should it implement a `Rolling` method?

Reply:
Added a `ArrowStringDataFrameColumnRollingWindow` now but it's super limited considering `ArrowStringDataFrameColumn` is immutable